### PR TITLE
Handle protobuf v26 breaking change, and grpcio 1.64.0 change

### PR DESCRIPTION
**Pull request by AkaZecik**
See the following for details:
- https://protobuf.dev/news/2024-02-05/
- https://github.com/grpc/grpc/issues/36683
- https://github.com/grpc/grpc/pull/36696

### CODE REVIEW

File: django_grpc_framework/test.py [hunk 1]
```diff
@@ -111,6 +111,11 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
         _validate_generic_rpc_handlers(generic_rpc_handlers)
         self.rpc_method_handlers.update(generic_rpc_handlers[0]._method_handlers)
 
+    def add_registered_method_handlers(self, service_name, method_handlers):
```
Comment (mlazowik) [inline django_grpc_framework/test.py:114]: Do I understand correctly that if we use `1.64.1` or newer we don't need this?

**Review by mlazowik — CHANGES_REQUESTED** [review]

File: django_grpc_framework/test.py (hunk 1 above)
Comment (AkaZecik) [reply django_grpc_framework/test.py:114]: I think we wouldn't need this if `FakeServer` was inheriting from `Server` class defined in `grpcio`.

The newest version of `grpcio` (`1.65.2`) adds a line `server.add_registered_method_handlers(...)` to the code generated from proto files, where in our case `server` is an instance of `FakeServer`, so the code simply crashes.

Comment (mlazowik) [reply django_grpc_framework/test.py:114]: nit: newest is `v1.65.4`

**Review by mlazowik — APPROVED** [review]

File: django_grpc_framework/test.py (hunk 1 above)
Comment (AkaZecik) [reply django_grpc_framework/test.py:114]: They are moving fast, it was the newest on Friday when I updated it :smile:

**Pull request merged**